Packs/ArcherRSA/Integrations/ArcherV2/ArcherV2.py

```diff
@@ -923,32 +923,36 @@ def get_field_value_list_helper(self, child, values_list, depth, parent="root"):
 
     def get_field_value_list(self, field_id, depth=0):
         cache = get_integration_context()
+        cache_key = f"{field_id}__{depth}"  # Need to set and get from cache using both field ID and depth
```

Comment:
Lets make the comment explain the depth part, as this is the same as the next comment :)

Reply:
Improved comment with better explanation